### ISSUE TITLE
Use local variables as objects in Java code

### ISSUE THREAD

**Issue by Matocolotoe** [btk5h/skript-mirror#119]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated not to be able to use custom variables as players in Java syntax.
I'll explain myself.

This works perfectly
`player.getWorld().getName()`

This doesn't work, though the local variable {_p} is a player
`{_p}.getWorld().getName()`

**Describe the solution you'd like**
It would be really nice, especially in functions, to be able to use local variables as objects as I mentioned previously.

**Describe alternatives you've considered**
Is it possible to do like below ?
`Player p = (Player) {_p}`

**Comment by abandonedaccount6235** [comment]
this is already possible exactly as you described

**Comment by btk5h** [comment]
Variables should work. Do you have code that can replicate this issue?

**Comment by Matocolotoe** [comment]
Yes I have @btk5h.

```
function test(p: player):
    send "%{_p}.getWorld().getName()%" to {_p}
```

Wait is it the world thing that is broken because of Skript ?

**Comment by Matocolotoe** [comment]
any news ?

**Comment by HAHAYOUDEAD** [comment]
> ```
> function test(p: player):
>     send "%{_p}.getWorld().getName()%" to {_p}
> ```
> 
> Wait is it the world thing that is broken because of Skript ?

it works for me on skript-mirror 1.0.0, paper-551, skript 2.3.5

**Comment by Matocolotoe** [comment]
lol broken because of skript itself but they won't fix it because even 1.8 and 1.12 are "outdated versions" that's why 3/5 of the overall servers are in these versions lmaoooooooo good strategy to get more visibility and satisfied players
